app/components/blacklight/document/action_component.rb

```diff
@@ -16,10 +16,28 @@ def initialize(document:, action:, options: {}, url_opts: {}, id: nil, link_clas
         @link_classes = link_classes
       end
 
+      def call
```

Comment:
By overriding `#call`, I think we're taking away the ability for action subclasses to use templates (because they'll inherit this `#call` method). Can we keep using the partial?

Reply:
Sounds good. I've updated to keep the template.